internal/server/statvar/statvar_match.go

```diff
@@ -35,7 +36,23 @@ func toQueryString(m map[string]string) string {
 	return b.String()
 }
 
-const defaultLimit = 100
+func buildQuery(in *pb.GetStatVarMatchRequest) query.Query {
+	var queries []query.Query
+	propertyValueQuery := toQueryString(in.GetPropertyValue())
+	if propertyValueQuery != "" {
+		query := bleve.NewMatchQuery(propertyValueQuery)
+		query.SetField("KeyValueText")
+		queries = append(queries, query)
+	}
+	if in.GetQuery() != "" {
+		query := bleve.NewMatchQuery(in.GetQuery())
+		query.SetBoost(1.5)
```

Comment:
comment on what is boost and why choose 1.5? Also make 1.5 constant?

Reply:
Exposed in the api.